### PR TITLE
VTJ-service + YTJ-kirjasto Java11

### PR DESCRIPTION
**Pull request by jakaarl**

### CODE REVIEW

**Review by mkamka — APPROVED** [review]
Ei valittamista

File: DEV-vtj-service.properties [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+vtj.production.env=false
+vtj.keystore.file=/huu/haa/keystore.jks
+vtj.keystore.password=huuhaa
+vtj.truststore.file=/huu/haa/truststore.jks
+vtj.truststore.password=huuhaa
+vtj.cert.alias=huuhaa
+vtj.username=huuhaa
+vtj.password=huuhaa
+vtj.address=http://huu.haa
+cas.service.vtj-service=http://huu.haa
```
Comment (jollem) [inline DEV-vtj-service.properties:11]: seems that one required property is missing here, namely _host.cas_
after adding `host.cas=http://huu.haa` here everything worked _smooooothly_

**Review by jollem — COMMENTED** [review]
one minor detail...

File: DEV-vtj-service.properties (hunk 1 above)
Comment (jollem) [reply DEV-vtj-service.properties:11]: user error.

**Review by jollem — APPROVED** [review]
👌

**Pull request merged**